spark/src/main/scala/org/apache/spark/sql/delta/PreDowngradeTableFeatureCommand.scala

```diff
@@ -211,7 +211,7 @@ case class TypeWideningPreDowngradeCommand(table: DeltaTableV2)
 
     recordDeltaEvent(
       table.deltaLog,
-      opType = "delta.typeWideningFeatureRemovalMetrics",
+      opType = "delta.typeWidening.featureRemovalMetrics",
```

Comment:
Nit: Just "delta.typeWidening.featureRemoval" or "delta.typeWidening.featureRemoval.metrics"

Reply:
Changed to `delta.typeWidening.featureRemoval`